explorer/websockethandlers.go

```diff
@@ -97,9 +140,54 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 
 				case "getmempooltxs":
 					exp.MempoolData.RLock()
-					msg, err := json.Marshal(exp.MempoolData)
+					mempoolData := exp.MempoolData
 					exp.MempoolData.RUnlock()
 
+					txCount := len(mempoolData.Transactions)
+					mempoolTxs := make([]*TxInfo, 0, txCount)
+					for _, tx := range mempoolData.Transactions {
+						exptx := exp.blockData.GetExplorerTx(tx.Hash)
```

Comment:
All this code reflects big design issues.  There should not be DB queries inside this websocket loop!  All of this code is done for each websocket client.

Note how previously it was just a `json.Marshal`.   The data was already there.  It needs to be again.